app/scripts/modules/kubernetes/src/v2/manifest/delete/deleteEntry.component.ts

```diff
@@ -0,0 +1,49 @@
+import { IComponentOptions, IController, module } from 'angular';
+
+import { IDeleteCommand } from './delete.controller';
+
+class KubernetesDeleteManifestCtrl implements IController {
+  public command: IDeleteCommand;
+}
+
+class KubernetesDeletManifestEntryComponent implements IComponentOptions {
```

Comment:
What's the intended name here? I don't know where Entry comes from, and I thought we were trying to avoid the use of Manifest?

Reply:
I was mimicking the naming you used here: https://github.com/spinnaker/deck/blob/master/app/scripts/modules/kubernetes/src/v2/manifest/wizard/manifestEntry.component.ts

Trying to keep it consistent.